src/dqe_idx.erl

```diff
@@ -227,6 +228,28 @@ metrics(Collection) ->
 
 %%--------------------------------------------------------------------
 %% @doc
+%% Returns a list of metric path suffixes of `Depth' that are prefixed
+%% by the given probe `Prefix', which can also be empty.
+%%
+%% For example:
+%% metrics(<<"collection">>, <<>>, 1) -> [<<"base">>].
+%% metrics(<<"collection">>, <<"base">>, 1) -> [<<"cpu">>].
+%% metrics(<<"collection">>, <<>>, 2) -> [[<<"base">>.<<"cpu">>]].
```

Comment:
Tiny thing but I thik this this needs to b a `,` not a `.` between `<<"base">>` and `<<"cpu">>`